magprime/automated_emails.py

```diff
@@ -7,6 +7,7 @@ class SeasonSupporterEmail(AutomatedEmail):
     def __init__(self, event):
         AutomatedEmail.__init__(self, SeasonPassTicket,
                                 subject='Claim your {} tickets with your MAGFest Season Pass'.format(event.name),
+                                ident='magprime_season_supporter_{}_invite'.format(event.name.lower().strip().replace(' ', '_')),
```

Comment:
This replacement makes me nervous, since it means that changing a season events' name will resend an email (and it's not obvious that it would do so unless you're looking at this exact line). This technically isn't any worse than it was before, but let's do this:
1. You should be able to go ahead and use the `event.slug` instead of `event.name` -- this should let you skip some of the string-mutating functions here, since slugs must be separated by underscores. This also lets us change an event's display name without re-sending emails.
2. In https://github.com/magfest/magprime/blob/master/magprime/configspec.ini, please add a comment noting that season_events' section names are tied to the emails we send out, and to be careful about adding, removing, or changing sections.